### PR TITLE
W-13114423-rtfctl-enhancements-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 1]
```diff
@@ -446,6 +446,34 @@ Generates a diagnostics report at `rtf-report.tar.gz`
 rtfctl report --namespace <rtf_namespace>
 ----
 
+Run rtfctl report tasks in parallel by using the `workers` flag that enables how many tasks to run in parallel. The  The default value is 10.
```
Comment (tasmoore) [inline modules/ROOT/pages/install-rtfctl.adoc]: There's an extra "The" in the second sentence.

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 2]
```diff
@@ -446,6 +446,34 @@ Generates a diagnostics report at `rtf-report.tar.gz`
...
+rtfctl report --workers 10
+----
+
+Generate rtfctl report compression faster by using the `use-fast-compression` flag that enables the compression of the result report archive faster in exchange to have a large report file size. The default value is false.
+
+----
+rtfctl report --use-fast-compression=true
+----
+
+Select which application namespaces you want to include in your report besides from the global rtf installation namespace by using the `<rtf_namespace>`. Using this flag enables you choose what to include in your reports, and returns a faster report.
```
Comment (tasmoore) [inline modules/ROOT/pages/install-rtfctl.adoc]: - Should "besides" be removed in the first sentence?
- Add "to" between "you choose" in the second sentence.

**Review by tasmoore — DISMISSED** [review]

**Review by tasmoore — DISMISSED** [review]

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 3]
```diff
@@ -446,6 +446,34 @@ Generates a diagnostics report at `rtf-report.tar.gz`
 rtfctl report --namespace <rtf_namespace>
 ----
 
+Run rtfctl report tasks in parallel by using the `workers` flag that enables how many tasks to run in parallel. The default value is 10.
+
+----
+rtfctl report --workers 10
+----
+
+Generate rtfctl report compression faster by using the `use-fast-compression` flag that enables the compression of the result report archive faster in exchange to have a large report file size. The default value is false.
```
Comment (a-palchikov) [inline modules/ROOT/pages/install-rtfctl.adoc]:
```suggestion
Faster compression of the tarball generated by `rtfctl report` with the `--use-fast-compression` flag. The default value is false.
```

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 4]
```diff
@@ -446,6 +446,34 @@ Generates a diagnostics report at `rtf-report.tar.gz`
...
+
+----
+rtfctl report --use-fast-compression=true
+----
+
+Select which application namespaces you want to include in your report besides from the global rtf installation namespace by using the `<rtf_namespace>`. Using this flag enables you to choose what to include in your reports, and returns a faster report.
```
Comment (a-palchikov) [inline modules/ROOT/pages/install-rtfctl.adoc]:
```suggestion
Select which application namespaces to include in the report besides the installation namespace by using the `--namespaces` flag. The flag controls what to include in the reports and can result in faster collection if only selected namespaces are reported. 
```

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 5]
```diff
@@ -446,6 +446,34 @@ Generates a diagnostics report at `rtf-report.tar.gz`
 rtfctl report --namespace <rtf_namespace>
 ----
 
+Run rtfctl report tasks in parallel by using the `workers` flag that enables how many tasks to run in parallel. The default value is 10.
+
+----
+rtfctl report --workers 10
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/install-rtfctl.adoc]: perhaps use a non default value for number of workers? `rtfctl report --workers 12`

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 6]
```diff
@@ -533,6 +561,15 @@ The `rtfctl` command line tool includes a last output line on the most verbose c
 
 If you are in doubt about whether a message represents a problem or not, discuss it with your system administrator or contact MuleSoft support.
 
+== Download Previous rtfctl CLI Versions
+
+Use the following API rest call to download different rtfctl tool CLI versions through the rtf management service API:
```
Comment (a-palchikov) [inline modules/ROOT/pages/install-rtfctl.adoc]:
```suggestion
Use the following rest API call to download different rtfctl tool CLI versions:
```

File: modules/ROOT/pages/install-rtfctl.adoc (hunk 5 above)
Comment (kmendiratta-crm) [reply modules/ROOT/pages/install-rtfctl.adoc]: or better yet just leave it as a numeric placeholder? `rtfctl report --workers <num_of_workers>`

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 7]
```diff
@@ -446,6 +446,34 @@ Generates a diagnostics report at `rtf-report.tar.gz`
 rtfctl report --namespace <rtf_namespace>
 ----
 
+Run rtfctl report tasks in parallel by using the `workers` flag that enables how many tasks to run in parallel. The default value is 10.
```
Comment (a-palchikov) [inline modules/ROOT/pages/install-rtfctl.adoc]:
```suggestion
Run rtfctl report tasks in parallel by using the `workers` flag that controls the number of tasks to run in parallel. The default value is 10.
```

**Review by tasmoore — APPROVED** [review]

**Review by kmendiratta-crm — APPROVED** [review]

**Pull request merged**